tests/tacacs/utils.py

```diff
@@ -22,6 +22,18 @@ def start_tacacs_server(ptfhost):
     ptfhost.command("service tacacs_plus restart", module_ignore_errors=True)
     return "tacacs+ running" in ptfhost.command("service tacacs_plus status", module_ignore_errors=True)["stdout_lines"]
 
+def stop_tacacs_server(ptfhost):
+    ptfhost.service(name="tacacs_plus", state="stopped")
+    check_all_services_status(ptfhost)
+
+def setup_local_user(duthost, creds_all_duts):
+    try:
+        duthost.shell("sudo deluser {}".format(creds_all_duts[duthost]['local_user']))
+    except:
```

Comment:
>except [](http://example.com/codeflow?start=4&length=6)

Could you use more specific exception type?

Reply:
Because this PR approved, so the fix will include in the per-command accounting test case PR.
Also this PR need wait bash plugin PR and sonic-utilities submodule update PR finish first.

Reply:
Sorry I mean "more specific exception type". Currently it catch all, even a non-exception type. Actually the `shell()` will possibly throw specific exception type.

Reply:
Fixed, change to catch RunAnsibleModuleFail